authexternalbrowser.go

```diff
@@ -282,7 +282,13 @@ func doAuthenticateByExternalBrowser(
 	}
 
 	if err = openBrowser(loginURL); err != nil {
-		return authenticateByExternalBrowserResult{nil, nil, err}
```

Comment:
What this really means: Instead of giving up, give the user the URL and see if they do the URL authentication, and the TCP listener will do the rest.

Reply:
That's a scenario I want to test. Can't be sure. I also think we have some ambiguity around the target behavior the customer wants to see / what dbt Python does / etc. I'll raise this in adapter meeting..